class/defaults.yml

```
@@ -1,3 +1,57 @@
 parameters:
   keycloak:
     namespace: syn-keycloak
+    # Hostname should be overwritten on the cluster level
+    hostname: keycloak.example.com
+    # Keycloak Admin
+    admin:
+      secretname: keycloak-admin-user
+      username: admin
+      password: '?{vaultkv:${customer:name}/${cluster:name}/keycloak/admin-password}'
+    # Ingress or Route should be enabled on the distribution level
+    ingress:
+      enabled: false
+    route:
+      enabled: false
+    # Labels can be extended in the config hierarchy by providing further
+    # entries in key `labels`.
+    labels:
+      app.kubernetes.io/name: keycloak
+      app.kubernetes.io/instance: syn-keycloak
+      app.kubernetes.io/version: v11.0.0
+      app.kubernetes.io/component: keycloak
+      app.kubernetes.io/managed-by: commodore
+    # Pod resource requests and limits
+    resources:
+      requests:
+        memory: "512Mi"
+        cpu: "500m"
+        limits:
+          memory: "1Gi"
+          cpu: "1"
+    # Extra java opts are appended to the default options set in
+    # `class/keycloak.yml`.
+    extraJavaOpts: ""
+    # Enable ServiceMonitor, PrometheusRule, and all Keycloak statistics on
+    # the metrics endpoint by default.
+    monitoring:
+      enabled: true
+      statistics: all
+    # Use Bitnami Postgres installed by the Keycloak chart by default
+    postgres:
+      builtin: false
```

According to the README the default is to deploy the builtin postgres.
```suggestion
      builtin: true
```

---

Yes, this is supposed to default to `true`, I forgot to revert the change after testing the external DB mode